src/jv_dtoa.c

```diff
@@ -1723,6 +1724,7 @@ gethex(struct dtoa_context* C,  CONST char **sp, U *rvp, int rounding, int sign)
 		  case '-':
 			esign = 1;
 			/* no break */
+			JQ_FALLTHROUGH;
```

Comment:
We no longer need `no break` comment?

Reply:
Well, this isn't our code, and the `no break` comments never had any effect as far as compilers and linters go, and I think leaving them there as a bread-crumb on what to look for in history might be useful.  Though, yeah, they serve no purpose now _and_ you can still find them in the history, so I could drop those.  Your call.

Reply:
Hmm, maybe we shouldn't edit jv_dtoa.c for any purpose.

Reply:
That is definitely an option, though it's been edited some -- first of all by @stedolan.

Reply:
I couldn't find the original base code of dtoa.c when included into the repository, but I think the original diff was only `jvp_` prefix and `MALLOC`/`FREE` definitions. The latest version is available at https://www.netlib.org/fp/ so I downloaded dtoa.c and g_fmt.c to compare against jv_dtoa.c, and there seems to be some bug fixes. I don't confirm jq will be affected by the fixes, but hopefully we could follow the upstream. So it's better to report to the original author (maybe along with 6151f204776db83047bfad1b76d5db0617a535f3), and then include the updates.

Reply:
I remember finding the original and seeing that @stedolan made some modifications to the default representation of numbers.  But it's been years since I looked.

Reply:
https://www.google.com/search?q=dtoa.c+david+m+gay